sympy/codegen/tests/test_matrix_nodes.py

```diff
@@ -0,0 +1,9 @@
+from sympy.core.symbol import symbols
+from sympy.matrices.dense import Matrix
+from sympy.codegen.matrix_nodes import MatrixSolve
+
+
+def test_matrix_solve_24862():
```

Comment:
I think explicitly saying issue in test names that reference a specific issue makes this clearer.
```suggestion
def test_matrix_solve_issue_24862():
```